appomni_appomni/README.md

```diff
@@ -0,0 +1,81 @@
+# AppOmni
+
+## Overview
+
+**AppOmni** is a leading SaaS security platform that provides deep visibility, monitoring, and control over enterprise SaaS applications. It helps organizations secure their data by detecting misconfigurations, unauthorized access, and potential threats in SaaS cloud applications.
+
+The **AppOmni integration** centralizes your SaaS security logs within Datadog, giving you a unified view of security events and alerts. AppOmni normalizes security events across different SaaS applications, ensuring consistency in log structure and making it easier to correlate activities across platforms.
+
+AppOmni generates alerts based on multiple detection methods:
+
+-   **User and Entity Behavior Analytics (UEBA):** Detects anomalies in user activity by establishing baselines and identifying deviations that may indicate compromised accounts or insider threats.
+-   **Threshold-based Alerts:** Triggers notifications when predefined limits (e.g., excessive failed login attempts, high-volume data exports) are exceeded.
+-   **Sequence-based Detection:** Identifies complex attack patterns by analyzing the order and relationship of events, such as privilege escalation followed by sensitive data access.
+
+By integrating with Datadog, these insights enable security teams to detect, investigate, and respond to threats in real time, improving the overall security posture of their SaaS environments.
+
+## Setup
+
+### Create a Datadog API key
+
+Create a Datadog [API Key][1].  See the steps below:
+
+Within Datadog, navigate to **Organization settings**, then click **API Keys**.
+
+1. Click **New Key**.
+
+2. Provide a name for the API key.
+
+3. Click **Copy API key**, and save this key for later.
+
+### Create a Datadog AppOmni Destination
+
+1.  Log into AppOmni
+
+2. Navigate to **Threat Detection** and select **Destinations**.
+
+3.  Click **Add New Destination**.
```

Comment:
Step 2 is displaying differently than the others, I think the extra spaces need to be removed in front of each step (in other words, use the same spacing everywhere that step 2 uses).

<img width="486" alt="Screenshot 2025-03-18 at 12 41 45 PM" src="https://github.com/user-attachments/assets/1d196a05-d0f9-4ed5-baa1-c4d39e9d25fd" />